install_caffe.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+sudo apt-get install protobuf-compiler libboost-all-dev libgflags-dev libgoogle-glog-dev libhdf5-serial-dev libleveldb-dev liblmdb-dev libsnappy-dev libopencv-dev libyaml-dev
+
+git clone https://github.com/BVLC/caffe.git ../caffe
+
+cp Makefile.config.example ../caffe/Makefile.config
+
+cd ../caffe
```

Comment:
(I made this script to be executed from the `dl-machine` folder. It would probably be more robust if I put absolute directories ... was my intention, but I forgot before making the PR)